dspace-api/src/main/java/org/dspace/discovery/configuration/DiscoveryConfiguration.java

```diff
@@ -37,6 +37,7 @@
     private DiscoverySortConfiguration searchSortConfiguration;
 
     private int defaultRpp = 10;
+    private int maxRpp = 20;
```

Comment:
after reconsidering it, IMHO we can simply set the default for maxRpp to 100 to respect the "default" behavior of the UI. If anyone has customized the UI it can easily adapt the max allowed value to his need with a quick look to the new configuration files